python/mxnet/ndarray.py

```diff
@@ -1239,21 +1247,23 @@ def add(lhs, rhs):
     # pylint: enable= no-member, protected-access
 
 def subtract(lhs, rhs):
-    """Subtracts arguments element-wise with broadcasting.
+    """Returns element-wise difference of the arguments with broadcasting.
 
-    Equivalent to ``lhs - rhs``.
+    Equivalent to ``lhs - rhs`` and ``mx.nd.broadcast_sub(lhs, rhs)``.
 
     Parameters
     ----------
     lhs : scalar or array
+        First array to be subtracted.
     rhs : scalar or array
-        The arrays to be added. If ``lhs.shape != rhs.shape``, they must be
+         Second array to be subtracted.
+        The arrays to be subtracted. If ``lhs.shape != rhs.shape``, they must be
```

Comment:
"Second array to be subtracted.
The arrays to be subtracted."

This is redundant, no? After introducing "First array to be subtracted." and "Second array to be subtracted." why do we then need to write "The arrays to be subtracted"?